.gitignore

```diff
@@ -12,4 +12,6 @@ bin/
 obj/
 wwwroot/
 /source/Native.Profiler/x64/
-__diff_output__/
\ No newline at end of file
+__diff_output__/
+.ionide/
+.idea/

```

Comment:
Is it from Rider?

Reply:
`.idea/`, yes. `.ionide/` is from [the F# vscode extension with the same name](https://ionide.io).